src/settings.ts

```diff
@@ -46,6 +46,9 @@ export const DEFAULT_SETTINGS: GrammarlySettings = {
     '[mdx]': {
       ignore: ['code'],
     },
+    '[asciidoc]': {
+      ignore: ['code'],
```

Comment:
`ignore` requires a parser to get token names in a given range. e.g. [markdown](https://github.com/znck/grammarly/blob/master/src/server/parsers/markdown.ts)